handler/OpenIDLoginHandler.inc.php

```diff
@@ -199,7 +199,7 @@ function signOut($args, $request)
 				}
 			}
 		}
-		$request->redirect(Application::get()->getRequest()->getContext(), 'index');
+		$request->redirect(Application::get()->getRequest()->getContext()->getPath(), 'index');
```

Comment:
OpenID is a site wide plugin. this means that it can be used as a Site wide login mechanism. In this case, the `Application::get()->getRequest()->getContext()->getPath()` will fail with a fatal error, if we use the site level logout mechanism. You can check this if you try the plugin in a multi-journal installation, and add it through the `Administration -> Site Seetings -> Plugins`

Reply:
Thanks for letting me know, the question is where to redirect when we getContext() is null?